operator/gitops/argocd/grafana/dashboards/pipeline-service-dashboard.json

```diff
@@ -87,14 +87,11 @@
         "y": 0
       },
       "id": 93,
+      "repeat": "datasource",
       "title": "Alerts",
       "type": "row"
     },
     {
-      "datasource": {
```

Comment:
I believe this was part of the problem with the Alert panels not rendoring immediately